setup.cfg

```diff
@@ -49,6 +49,7 @@ install_requires =
     mccabe>=0.6,<0.7
     toml>=0.7.1
     colorama;sys_platform=="win32"
+    typing-extensions>=3.10.0;python_version<"3.10"
```

Comment:
👍

Reply:
Necessary to be able to import `TypeGuard`